### PR TITLE
:white_check_mark: Add helpers to build files for testing

### PR DESCRIPTION
**Pull request by hirunatan**
Refactor tests of common module, to make use of a new suite of helpers to build files.

### CODE REVIEW

File: common/test/common_tests/helpers/compositions.cljc [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+      (thf/add-sample-shape child-label
+                            :type :rect
+                            :name "Rect1"
+                            :parent-label root-label)
+      (thf/make-component component-label root-label)))
+
+(defn add-simple-component-with-copy
+  [file component-label main-root-label main-child-label copy-root-label]
+  (-> file
+      (thf/add-sample-shape main-root-label
```
Comment (pabloalba) [inline common/test/common_tests/helpers/compositions.cljc]: It should call add-simple-component

File: common/test/common_tests/helpers/compositions.cljc [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+                            :name "Frame1")
+      (thf/add-sample-shape child-label
+                            :type :rect
+                            :name "Rect1"
+                            :parent-label frame-label)))
+
+(defn add-simple-component
+  [file component-label root-label child-label]
+  (-> file
+      (thf/add-sample-shape root-label
```
Comment (pabloalba) [inline common/test/common_tests/helpers/compositions.cljc]: It should call add-frame-with-child

File: common/test/common_tests/helpers/compositions.cljc (hunk 1 above)
Comment (hirunatan) [reply common/test/common_tests/helpers/compositions.cljc]: done

File: common/test/common_tests/helpers/compositions.cljc (hunk 2 above)
Comment (hirunatan) [reply common/test/common_tests/helpers/compositions.cljc]: done

**Pull request merged**